### PR TITLE
Implemented the E2E test cases to show the overlap VRT for Add-Customer/Edit-Customer/Add-User/Edit-User

### PR DESCRIPTION
**Pull request by robinr**
Following can be observed the Overlap widgets 
a) Add-Customer
b) Edit-Customer
c) Add-User
d) Edit-User

Now its in the centre
The E2E testcases shows the overlap on top of SettingsForm that of Customer/User.

### CODE REVIEW

**Comment by robinr** [comment]
From my observation, there is some mis-match between the C9 and the CI environment, as the failure is not not getting re-produced on the C9.

**Comment by robinr** [comment]
settings-form-with-add-customer-actual-diff
settings-form-with-add-user-actual-diff
settings-form-with-edit-customer-actual-diff
settings-form-with-edit-user-actual-diff

The observed problems are specific to CI environment..(Above obtained from the investigation of the logs if image diffs).

**Comment by robinr** [comment]
Latest commit is rebased with the development and verified and its good for a merge.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Changelog for Mantarray Frontend Components
 ===========================================
+0.1.9 (2020-12-22)
```
Comment (eli88fine) [inline CHANGELOG.rst:3]: until we actually publish a new version, please just put `0.1.9 (unreleased)`

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+------------------
+
+- Added the File Upload Widget and SettingsForm E2E testing VRT to validate Overlap for the dialog of Add/Edit options for Customer and User
```
Comment (eli88fine) [inline CHANGELOG.rst:6]: The File Upload Widget and the Settings Form are very different features, so please have them on separate lines of the ChangeLog

**Review by eli88fine — COMMENTED** [review]
It looks like this branch is still out-of-date with `development`.  Also, please add some E2E tests for the dialog closing (in various ways--Save / Delete / Cancel) to confirm that integration of the two components (SettingsForm.vue and the PopUpDialog Components) is successful

File: CHANGELOG.rst (hunk 1 above)
Comment (robinr) [reply CHANGELOG.rst:3]: Modified.

File: CHANGELOG.rst (hunk 2 above)
Comment (robinr) [reply CHANGELOG.rst:6]: File Upload Widget, SettingsForm VRT are separated.

**Comment by robinr** [comment]
Testcafe works only pre-defined available DOM elements which are available at page load time, Selectors uses the class or id names to choose simulate user actions, I have tried trying to use Selector  on dynamically attached dialog's, that gets attached during the button function execution, during Add Customer/Edit Customer or for Add User/Edit User, this will result in selection error exception as the class names and id names keep changing for every execution, so we cannot do any further verification like Cancel / Add/ Delete actions. 

If you have any prototypes of such testcases which is able to work on runtime using selectors on PopUpDialogs, which would allow selection, input edit, and click on dynamic classes or id's then we can try, for now I think its not feasible. As this doesn't work.

**Review by eli88fine — APPROVED** [review]

**Pull request merged**